vector/src/main/java/im/vector/app/core/preference/PushRulePreference.kt

```diff
@@ -130,10 +130,11 @@ class PushRulePreference : VectorPreference {
                 }
             } else {
                 if (NOTIFICATION_OFF_INDEX == index) {
-                    if (safeKind == RuleSetKey.UNDERRIDE || safeRule.ruleId == RuleIds.RULE_ID_SUPPRESS_BOTS_NOTIFICATIONS) {
+                    if (safeKind == RuleSetKey.UNDERRIDE && safeRule.ruleId != RuleIds.RULE_ID_CALL) {
```

Comment:
I understand that it was not possible for `safeRule.ruleId == RuleIds.RULE_ID_SUPPRESS_BOTS_NOTIFICATIONS` to be true here. But why checking `RULE_ID_CALL` now?

To be honest, this code is really hard to follow, it misses some comment... :/